src/Controller/JobController.php

```diff
@@ -39,4 +39,60 @@ public function discoverJobs() {
     return $this->redirect('entity.ultimate_cron_job.collection');
   }
 
+  /**
+   * Displays a detailed cron job logs table.
+   *
+   * @param \Drupal\ultimate_cron\Entity\CronJob $ultimate_cron_job
+   *   The cron job which will be run.
+   *
+   * @return array
+   *   A render array as expected by drupal_render().
+   */
+  public function showLogs(CronJob $ultimate_cron_job) {
+
+    $rows = array();
+    $header = array(
+      $this->t('Severity'),
+      $this->t('User'),
+      $this->t('Name'),
+      $this->t('Lid'),
+      $this->t('Start Time'),
+      $this->t('End Time'),
+      $this->t('Init Message'),
+      $this->t('Message'),
```

Comment:
not sure about showing init message and message. Maybe have a single message field that shows both if available?